openmc/examples.py

```diff
@@ -0,0 +1,619 @@
+    clad_cell = openmc.Cell(name='clad', fill=clad, region=+fuel_or & -clad_or)
+    hot_water_cell = openmc.Cell(name='hot water', fill=hot_water, region=+clad_or)
+    fuel_pin_universe.add_cells([fuel_cell, clad_cell, hot_water_cell])
+
+
+    # Create a control rod guide tube universe
+    guide_tube_universe = openmc.Universe(name='Guide Tube')
+    gt_inner_cell = openmc.Cell(name='guide tube inner water', fill=hot_water,
+                                region=-fuel_or)
+    gt_clad_cell = openmc.Cell(name='guide tube clad', fill=clad,
+                               region=+fuel_or & -clad_or)
+    gt_outer_cell = openmc.Cell(name='guide tube outer water', fill=hot_water,
+                                region=+clad_or)
+    guide_tube_universe.add_cells([gt_inner_cell, gt_clad_cell, gt_outer_cell])
+
+    # Create fuel assembly Lattice
+    assembly = openmc.RectLattice(name='Fuel Assembly')
+    assembly.pitch = (pitch/17, pitch/17)
+    assembly.lower_left = (-pitch/2, -pitch/2)
+
+    # Create array indices for guide tube locations in lattice
+    template_x = np.array([5, 8, 11, 3, 13, 2, 5, 8, 11, 14, 2, 5, 8,
+                           11, 14, 2, 5, 8, 11, 14, 3, 13, 5, 8, 11])
+    template_y = np.array([2, 2, 2, 3, 3, 5, 5, 5, 5, 5, 8, 8, 8, 8,
+                           8, 11, 11, 11, 11, 11, 13, 13, 14, 14, 14])
+
+    # Create 17x17 array of universes
+    assembly.universes = np.tile(fuel_pin_universe, (17, 17))
+    assembly.universes[template_x, template_y] = guide_tube_universe
+
+    # Create root Cell
+    root_cell = openmc.Cell(name='root cell', fill=assembly)
+    root_cell.region = +min_x & -max_x & +min_y & -max_y
+
+    # Create root Universe
+    model.geometry.root_universe = openmc.Universe(universe_id=0, name='root universe')
+    model.geometry.root_universe.add_cell(root_cell)
+
+    model.settings.batches = 10
+    model.settings.inactive = 5
+    model.settings.particles = 100
+    model.settings.source = openmc.Source(space=openmc.stats.Box(
+        [-pitch/2, -pitch/2, -1], [pitch/2, pitch/2, 1], only_fissionable=True))
+
+    plot = openmc.Plot()
+    plot.origin = (0.0, 0.0, 0)
+    plot.width = (21.42, 21.42)
+    plot.pixels = (300, 300)
+    plot.color_by = 'material'
+    model.plots.append(plot)
+
+    return model
+
+
+def slab_mg(reps=None, as_macro=True):
+    """Create a one-group, 1D slab model.
+
+    Parameters
+    ----------
+    reps : list, optional
+        List of angular representations. Items can be 'ang', 'ang_mu', 'iso', or
+        'iso_mu'.
+    as_macro : bool, optional
+        Whether :class:`openmc.Macroscopic` is used
+
+    Returns
+    -------
+    model : openmc.model.Model
+        One-group, 1D slab model
+
+    """
+    model = openmc.model.Model()
+
+    # Define materials needed for 1D/1G slab problem
+    mat_names = ['uo2', 'clad', 'lwtr']
+    mgxs_reps = ['ang', 'ang_mu', 'iso', 'iso_mu']
+
+    if reps is None:
+        reps = mgxs_reps
```

Comment:
This is a case where the user and test needs differ.  If a user was using this for fun, they probably would just want all mgxs_reps values to be 'iso'.  But for the test of course we dont to touch more code.